diff --git a/node_modules/@synthetixio/synpress-metamask/dist/playwright/index.js b/node_modules/@synthetixio/synpress-metamask/dist/playwright/index.js
index 0eb0f5f..bd84e41 100644
--- a/node_modules/@synthetixio/synpress-metamask/dist/playwright/index.js
+++ b/node_modules/@synthetixio/synpress-metamask/dist/playwright/index.js
@@ -211,7 +211,7 @@ var activityTab = {
 var singleToken = ".multichain-token-list-item";
 var HomePage_default = {
   logo: `button${createDataTestSelector("app-header-logo")}`,
-  copyAccountAddressButton: createDataTestSelector("address-copy-button-text"),
+  copyAccountAddressButton: createDataTestSelector("app-header-copy-button"),
   currentNetwork: `${createDataTestSelector("network-display")} span:nth-of-type(1)`,
   threeDotsMenu,
   settings: settings_default,
@@ -1857,7 +1857,7 @@ var MetaMask = class extends MetaMaskAbstract {
     await this.homePage.closeTransactionDetails();
   }
 };
-var DEFAULT_METAMASK_VERSION = "11.9.1";
+var DEFAULT_METAMASK_VERSION = "11.16.7";
 var EXTENSION_DOWNLOAD_URL = `https://github.com/MetaMask/metamask-extension/releases/download/v${DEFAULT_METAMASK_VERSION}/metamask-chrome-${DEFAULT_METAMASK_VERSION}.zip`;
 async function prepareExtension(forceCache = true) {
   let outputDir = "";
